### PR TITLE
Add latest updates on dashboard

### PR DESCRIPTION
**Pull request by lickem22**
Reviewer: @sidravi1 
Estimate: 20mins

---

### Goal
The goal is to add the latest updates blogpost
### Changes
- Added a latest updates blogpost for the question dashboard.
- Added a `I have added blogpost in Latest Updates` field in the pull request template

## Checklist

Fill with `x` for completed. Delete any lines that are not relevant

- [ ] My code follows the style guidelines of this project
- [ ] I have reviewed my own code to ensure good quality
- [ ] I have tested the functionality of my code to ensure it works as intended
- [ ] I have resolved merge conflicts
- [ ] I have updated the automated tests (if applicable)
- [ ] I have updated the requirements (if applicable)
- [ ] I have updated the README file (if applicable)
- [ ] I have updated affected documentation (if applicable)

### CODE REVIEW

File: docs/blog/posts/question-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+authors:
+  - Sid
```
Comment (suzinyou) [inline docs/blog/posts/question-dashboard.md:3]: Carlos?

File: docs/blog/posts/question-dashboard.md
Comment (suzinyou) [inline docs/blog/posts/question-dashboard.md:1]: Would be nice to add dashboard screen shot :)

**Comment by suzinyou** [comment]
Also just a question as I've not been following the dashboard thread..

I don't see a dashboard page on the docs! It says coming soon. Can you update this also?
https://idinsight.github.io/aaq-core/components/admin-app/

File: docs/blog/posts/question-dashboard.md (hunk 1 above)
Comment (lickem22) [reply docs/blog/posts/question-dashboard.md:3]: My bad. Copy/Paste gone wrong haha

**Review by suzinyou — APPROVED** [review]
I still think we should add a page for Dashboard on the docs since it's a nice feature, but I'll just approve this one for now. Created a ticket: https://idinsight.atlassian.net/browse/AAQ-537

**Pull request merged**